back/model/Status.go

```diff
@@ -0,0 +1,9 @@
+package model
+
+import "gorm.io/gorm"
+
+type Status struct {
+	gorm.Model
+	Name    string `json:"name"`
+	IsOrder bool   `json:"isOrder" default:"false"`
+}
```

Comment:
What is this model? What's its purpose?

Reply:
It's to know if the the cart is the current cart of the use or a previous order. It allows to store everything in the same table

Reply:
Would have it been better to have a column isOrder in every cart?

Reply:
The issue is the name: `Status`.

Status of what?

Reply:
It's the status of the order it could be:
- Cart
- Preparation in progress
- Delivery in progress
- Delivered
- Cancelled

Reply:
Then name it so! `OrderStatus`.

Moreover, if it's "just an enum", then why is it a full fledged entity?